UnityAds/src/main/java/com/mopub/mobileads/UnityAdsAdapterConfiguration.java

```diff
@@ -70,34 +104,34 @@ public void initializeNetwork(@NonNull final Context context, @Nullable final Ma
         synchronized (UnityAdsAdapterConfiguration.class) {
             try {
                 if (UnityAds.isInitialized()) {
-                    sendInitializationSuccess(listener, LOG_ALREADY_INITIALIZED);
+                    sendInitializationSuccess(listener, UnityAdsConstants.LOG_ALREADY_INITIALIZED.getMessage());
```

Comment:
Since you create this nice enum, now you could make reuse it.
For example `sendInitializationSuccess` and  `sendInitializationFailure` are the same in terms of flow: Call log and call listener with same same method. So what if we pass have smth like 
```java
private void sendInitializationResult(@NonNull final OnNetworkInitializationFinishedListener listener, final UnityAdsConstants result) {
    MoPubLog.log(CUSTOM, ADAPTER_NAME, result.getMessage());
    listener.onNetworkInitializationFinished(UnityAdsAdapterConfiguration.class,   result.getInitCode());
}
```

thoughts?

Reply:
Good point - implemented it